src/App.tsx

```diff
@@ -38,6 +38,7 @@ type Props = {
     expanded: boolean;
     onPermalinkClick: (commentId: number) => void;
     apiKey: string;
+    onHeightChange: () => void;
```

Comment:
Should it be mandatory?

Reply:
Tricky question. 

No. If we want discussion to work in other contexts
Yes. If we assume discussion is entwined with DCR

I guess my head is assuming the entwine but we should strive for the separation. I'll update